code/modules/materials/material_sheets.dm

```diff
@@ -9,23 +9,27 @@
 	max_amount = 60
 	randpixel = 3
 	icon = 'icons/obj/materials.dmi'
-	matter = null
+	matter = null // Material stacks are expected to populate their own matter var as they are generally abstract 'shapes' rather than discrete functional items.
```

Comment:
I know they don't work right now but any chance this could be a `//!` doc comment?

Reply:
It makes the comment show up when hovering over the item.
![image](https://user-images.githubusercontent.com/12163281/201475779-ee9ee14a-5a9e-4934-9fef-e958e01d78c6.png)
![image](https://user-images.githubusercontent.com/12163281/201475812-e7ef3df1-325b-4f1c-84e9-fc55d19cbfb8.png)

Reply:
Does not appear done.